### DISCUSSION TITLE
Creating Rule that is between June and August

### DISCUSSION THREAD

**Discussion by Barrow1990** [firefly-iii/firefly-iii#9230] [Firefly III] [answered]
Hi All,

The specific scenario is I have 2 cars which get there MOT done at the same garage.
The price is generally the cost of an MOT so Ill have the same transaction for each.
What I want to do is create a rule that assigns it to a specific bill.

The triggers im looking at are:
Description_Starts = 'PLACE'
Transaction date is before = August
Transaction date is after = June

This then creates an action to link_to_bill.

Note: The transaction date at a maximum detail can only be MM-DD. I want it to dynamically compensate for the year

**Comment by JC5** [comment, marked as answer]
Hi, great question!

https://docs.firefly-iii.org/references/firefly-iii/rule-triggers/

I haven't tested this, but "xxxx-07-01" should do what you need.

**Reply by Barrow1990** [reply]
Perfect thank you. I was missing that I required to use xxxx. I was using YYYY
@JC5 Maybe worth changing that logic to industry standard of YYYY-MM-DD and any other permutations of that.
